src/fibers/context/execution_context.cpp

```diff
@@ -0,0 +1,24 @@
+#include "yaclib/fibers/detail/execution_context.hpp"
+// TODO(myannyax) if
+#include "setup_stack_x64.hpp"
+
+extern "C" void yaclib_switch_context(void* from_context, void* to_context);
+
+namespace yaclib {
+
+ExecutionContext::ExecutionContext() : _context(new void*[kAsmContextSize]) {
```

Comment:
- [ ] Remove allocation.

Btw wrong size, you alloc 8*size, where size is count of registers * 8

Use char[kAsmContextSize]; and cast it to void**